emsdk.py

```diff
@@ -294,11 +294,11 @@ def remove_readonly_and_try_again(func, path, exc_info):
 def import_pywin32():
   if WINDOWS:
     try:
-      import win32api
-      import win32con
-      return win32api, win32con
-    except Exception:
-      exit_with_error('Failed to import Python Windows extensions win32api and win32con. Make sure you are using the version of python available in emsdk, or install PyWin extensions to the distribution of Python you are attempting to use. (This script was launched in python instance from "' + sys.executable + '")')
+        import winreg
+    except ImportError:
+        import _winreg as winreg
```

Comment:
Can you add a comment saying something like "python 2 support"... or "old python 2 name"

Reply:
```suggestion
		# old python 2 name
        import _winreg as winreg
```